README.md

```diff
@@ -223,6 +233,28 @@ these are configured in your Heroku app:
 - `BOT_NAME` (defaults to `coqbot`)
 - `BOT_EMAIL` (defaults to `BOT_NAME@users.noreply.github.com`)
 
+In the next release of coqbot, the `BOT_NAME` and `BOT_EMAIL`
+environment variables won't have any effect and should be instead
+set from a configuration file (see [`coqbot.toml`](coqbot.toml)).
+The port number must not be set in the configuration file if you're
+deploying the docker image to Heroku, since it uses a custom
+environment variable.
+
+A Dockerfile to build a personalized image based on a release image
+from GitHub packages, using a custom `bot_config.toml` configuration
+file would look like:
+```dockerfile
+FROM docker.pkg.github.com/coq/bot/coqbot:xxx
+
+COPY bot_config.toml ./
+
+EXPOSE 8000 # The port you specified in bot_config.toml (this command is ignored if you deploy to Heroku)
```

Comment:
Because if we comment out the port variable, our instance of the server won't read the port environment variable Heroku is providing. I'm going to document all this and prepare an example configuration file.